meteor/sofie-server/data-access/repositories/interfaces/part-repository.ts

```diff
@@ -2,4 +2,5 @@ import { Part } from '../../../model/entities/part'
 
 export interface PartRepository {
 	getParts(segmentId: string): Promise<Part[]>
+	deleteParts(segmentId: string): Promise<void>
```

Comment:
I really like that suggestion. That change explains really nicely why the argument name is what it is.
Also makes it clear that the method only deletes e.g segments that are children of the supplied rundown id.

Suggested change has been applied.

Reply:
This kinda depends on your naming "philosophy".

There is one philosophy where everything has to be stated in the name of the method i.e. `deleteSegmentParts` or `deletePartsForSegments`.

Another philosophy is that the paramenters is a part of the name of the method, i.e. `deleteParts(segmentId: string)` indicates that we are deleting Parts that have a relation to the parsed SegmentId.

I am fine with either "philosophy". I will however suggest that if you go with the first, I will recommed `deletePartsForSegment` instead of `deleteSegmentParts`.
The later implies that we have an entity called `SegmentPart` which is a concept we do not have.

Reply:
The methods have been renamed to follow Lindved's recommende naming scheme (delete<children>For<parent>).

Reply:
I think the _everything-in-the-method-name_ is the best philisophy for us, since function/method overloading is a bit of a mess in TypeScript.